packs/boons-and-curses/achaekek-minor-boon.json

```diff
@@ -4,7 +4,7 @@
     "name": "Achaekek - Minor Boon",
     "system": {
         "description": {
-            "value": "<p>Once, when you would fail a Stealth check, you critically succeed instead. Achaekek typically grants this boon for an extremely consequential Stealth check that could lead to an assassination, such as one that could get you into position to kill an important target, but rarely on a Stealth check to help you escape.</p>"
+            "value": "<p>Once, when you would fail a Stealth check, you critically succeed instead.</p>\n<p>Achaekek typically grants this boon for an extremely consequential Stealth check that could lead to an assassination, such as one that could get you into position to kill an important target, but rarely on a Stealth check to help you escape.</p>"
```

Comment:
Can you make a roll note for this? Sneak/Hide are secret actions, so the Note should only show up for the GM, but they should be notified that the boon could activate.